test/run-tests

```diff
@@ -328,6 +329,25 @@ class Task:
     def __str__(self):
         return f"task {self.owner}/{self.repo}/{self.pull}:{self.image['name']}"
 
+    def get_url(self):
+        source = self.image.get("source")
+        if source:
+            return source
+        compose_url = self.image.get("compose")
+        if compose_url:
+            variant = self.image.get("variant")
+            image_urls = composeurl2images(compose_url, "x86_64", variant)
+            if len(image_urls) == 1:
+                return image_urls[0]
+            else:
+                if image_urls:
+                    logging.error(
+                        f"ERROR: Multiple images found: {image_urls}"
+                        "in compose {compose_url}"
+                    )
+                else:
+                    logging.error(f"ERROR: no image found in compose {compose_url}")
```

Comment:
do we need to check in case there is no `source` and there is no `compose`?

Reply:
or is that handled below with the
```python
            if not image_url:
                return None
```
?

Reply:
That was indeed the idea. But a more explicit error message in the log would not hurt.

Reply:
fixed by rebase